client/src/gui/lobby/tracker.rs

```diff
@@ -31,15 +28,14 @@ impl<'a> Widget for Tracker<'a> {
             ui.allocate_exact_size(desired_size, Sense::focusable_noninteractive());
 
         for spat in Spatula::iter() {
-            let color = if let Some(Some(i)) = self.game.spatulas.get(&spat) {
-                self.players
-                    .get(i)
-                    .map(|p| p.options.color())
-                    .unwrap_or_default()
-            } else {
-                Color32::from_rgb(50, 50, 50)
-            };
-
+            let mut spat_state = self
+                .game
+                .spatulas
+                .get(&spat)
+                .unwrap_or(&SpatulaState::default())
+                .clone();
+            let color = spat_state.tier.get_color();
+            let color = Color32::from_rgb(color.0, color.1, color.2);
```

Comment:
We can probably just store the color as a `Color32` directly on the spatula state to clean up the gui code here since it impls `Serialize`/`Deserialize`. I'm not sure why I didn't do that for player colors. Fine to just leave this as-is though since spatula coloring logic will change in a follow-up PR according to #57